src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");

```

Comment:
The message should be more informative and indicate that it's a Bulldozer that started working. Consider including the class name in the message.